odoo/models.py

```diff
@@ -903,7 +904,7 @@ def flush(xml_id=None):
                     cr.execute('ROLLBACK TO SAVEPOINT model_load_save')
                 except Exception as e:
                     info = rec_data['info']
-                    message = (_(u'Unknown error during import:') + u' %s: %s' % (type(e), e))
+                    message = (_(u'Unknown error during import:') + u' %s: %s' % (type(e), traceback.format_exc()))
```

Comment:
Please don't drop `e`, as it may be a readable message for the user.
IMHO, you should show the triple `(type(e), e, traceback.format_exc())`.

By the way, the "Unknown error during import" is idiotic.
The error is known, it is simply not a database error.